### PR TITLE
[PID/(Q)EAA - SD-JWT parameters ] trust chain

### PR DESCRIPTION
**Pull request by SaraConsoliACN**
This PR aims to resolve the issue https://github.com/italia/eudi-wallet-it-docs/issues/259

### CODE REVIEW

File: docs/en/trust.rst [hunk 1]
```diff
@@ -627,6 +627,15 @@ The Wallet Instance that verifies the request issued by the Relying Party MUST u
 
 Furthermore, the Wallet Instance applies the metadata policy, if any.
 
+Trust Chain Fast Renewal
+------------------------
+
+After detailing the federation discovery process, we now turn our attention to the Trust Chain Fast Renewal method. This approach offers a streamlined way to maintain the validity of a trust chain without undergoing the full discovery
+process again. It's particularly useful for quickly updating trust relationships when minor changes occur or when the
+trust chain is close to expiration but the overall structure of the federation hasn't changed significantly.
```
Comment (peppelinux) [inline docs/en/trust.rst]:
```suggestion
Trust Chain is close to expiration but the overall structure of the federation hasn't changed significantly.
```

File: docs/en/trust.rst [hunk 2]
```diff
@@ -627,6 +627,15 @@ The Wallet Instance that verifies the request issued by the Relying Party MUST u
...
+
+The Trust Chain Fast Renewal process is initiated by fetching the leaf's (RP's or OP's) entity configuration anew. However, unlike the federation discovery process that may involve fetching entity configurations starting from the authority hints, the fast renewal focuses on directly obtaining the subordinate statements. These statements are requested using the source_endpoint provided within them, which points to the location where the statements can be fetched.
```
Comment (peppelinux) [inline docs/en/trust.rst]:
```suggestion
The Trust Chain fast renewal process is initiated by fetching the leaf's (RP's or OP's) Entity Configuration anew. However, unlike the federation discovery process that may involve fetching Entity Configurations starting from the authority hints, the fast renewal focuses on directly obtaining the Subordinate Statements. These statements are requested using the `source_endpoint` provided within them, which points to the location where the statements can be fetched.
```

File: docs/en/trust.rst [hunk 3]
```diff
@@ -627,6 +627,15 @@ The Wallet Instance that verifies the request issued by the Relying Party MUST u
 
 Furthermore, the Wallet Instance applies the metadata policy, if any.
 
+Trust Chain Fast Renewal
+------------------------
+
+After detailing the federation discovery process, we now turn our attention to the Trust Chain Fast Renewal method. This approach offers a streamlined way to maintain the validity of a trust chain without undergoing the full discovery
```
Comment (fmarino-ipzs) [inline docs/en/trust.rst]:
```suggestion
This Section outlines a Trust Chain fast renewal method. This approach offers a streamlined way to maintain the validity of a trust chain without undergoing the full discovery
```

Comment (fmarino-ipzs) [reply docs/en/trust.rst]: It is better to use impersonal forms without terms such as 'we', 'our', etc.

File: docs/en/trust.rst [hunk 4]
```diff
@@ -627,6 +627,15 @@ The Wallet Instance that verifies the request issued by the Relying Party MUST u
...
 Furthermore, the Wallet Instance applies the metadata policy, if any.
 
+Trust Chain Fast Renewal
+------------------------
+
+This Section outlines a Trust Chain fast renewal method. This approach offers a streamlined way to maintain the validity of a trust chain without undergoing the full discovery
+process again. It's particularly useful for quickly updating trust relationships when minor changes occur or when the
+Trust Chain is close to expiration but the overall structure of the federation hasn't changed significantly.
+
+The Trust Chain fast renewal process is initiated by fetching the leaf's (RP's or OP's) Entity Configuration anew. However, unlike the federation discovery process that may involve fetching Entity Configurations starting from the authority hints, the fast renewal focuses on directly obtaining the Subordinate Statements. These statements are requested using the `source_endpoint` provided within them, which points to the location where the statements can be fetched.
```
Comment (peppelinux) [inline docs/en/trust.rst]:
```suggestion
The Trust Chain fast renewal process is initiated by fetching the leaf's Entity Configuration anew. However, unlike the federation discovery process that may involve fetching Entity Configurations starting from the authority hints, the fast renewal focuses on directly obtaining the Subordinate Statements. These statements are requested using the `source_endpoint` provided within them, which points to the location where the statements can be fetched.
```

File: docs/en/trust.rst [hunk 5]
```diff
@@ -627,6 +627,15 @@ The Wallet Instance that verifies the request issued by the Relying Party MUST u
 
 Furthermore, the Wallet Instance applies the metadata policy, if any.
 
+Trust Chain Fast Renewal
+------------------------
+
+This Section outlines a Trust Chain fast renewal method. This approach offers a streamlined way to maintain the validity of a trust chain without undergoing the full discovery
```
Comment (peppelinux) [inline docs/en/trust.rst]:
```suggestion
The Trust Chain fast renewal method offers a streamlined way to maintain the validity of a trust chain without undergoing the full discovery
```

**Review by peppelinux — APPROVED** [review]

**Review by fmarino-ipzs — APPROVED** [review]

**Review by grausof — APPROVED** [review]

**Comment by SaraConsoliACN** [comment]
Fixes italia/eudi-wallet-it-docs#259

**Pull request merged**